.rubocop.yml

```diff
@@ -49,6 +50,186 @@ Style/RedundantFetchBlock:
     - 'app/controllers/concerns/emit_request_events.rb'
     - 'app/models/concerns/entity_events.rb'
 
+RSpec/NoShouldInSpecs:
+  Enabled: true
+  Exclude:
+  - 'spec/smoke/candidate_login_spec.rb'
```

Comment:
Humm I don't think I agree for us to exclude all this specs. I think we should renamed them to not become a two pattern specs from now on. Some using the should and others don't.

Reply:
We haven't had a rule to include should so we already have a mixure of "should" and "no should". I've never written a method with the word should in it, it's not a codified pattern so I don't think this is an issue.

Reply:
Ok. Can you create a card then to excludes when we edit it and fix the violations, please?